### DISCUSSION TITLE
Single-/Multithreaded

### DISCUSSION THREAD

**Discussion by Uatschitchun** [jellyfin/jellyfin#5078] [Q&A]
Hi there. Someone able to answer, what parts of jellyfin are single threaded and what parts are multi threaded?

Running jellyfin/jellyfin docker and I'm noticing its always only using one core whereas 6 (i5) are available. Docker isn't resource limited in any way, so if there are any operations that are able to run multi threaded, they should, shouldn't they?

Thx

**Comment by hurricane-dorian** [comment]
I'm not a jellyfing developer so I don't have an actual answer to your question but Jellyfin itself is pretty efficient. My lowly J4105 can easily handle it. ffmpeg transcoding is about the only time the system gets heavily taxed (ffmpeg is multithreaded by default) and enabling Intel QuickSync hardware transcoding greatly helps with speed/load/efficiency.
